meteor/lib/collections/PeripheralDevices.ts

```diff
@@ -145,6 +145,15 @@ export function getExternalNRCSName(device: PeripheralDevice | undefined): strin
 				return 'iNews'
 			} else if (device.type === PeripheralDeviceAPI.DeviceType.SPREADSHEET) {
 				return 'Google Sheet'
+			} else if (
+				device.type === PeripheralDeviceAPI.DeviceType.PLAYOUT  ||
+				device.type === PeripheralDeviceAPI.DeviceType.MEDIA_MANAGER ||
+				device.type === PeripheralDeviceAPI.DeviceType.PACKAGE_MANAGER
+			) {
+				// These aren't ingest gateways
+				return 'N/A'
+			} else {
+				assertNever(device.type)
```

Comment:
Considering that it's possible to connect to Sofie using a custom peripheralDevice, even if the typeguard is useful, it should still try and return some useful value, perhaps `Unknown: ${device.type}`?

Reply:
Good insight! Will fix!